protocol/chat.go

```diff
@@ -33,7 +33,10 @@ const (
 	ChatTypeCommunityChat
 )
 
-const pkStringLength = 68
+const (
+	timelineChatID = "@timeline70bd746ddcc12beb96b2c9d572d0784ab137ffc774f5383e50585a932080b57cca0484b259e61cecbaa33a4c98a300a"
```

Comment:
What is this magical string?

Reply:
A comment would be nice explaining the purpose.

Reply:
done, is a magical string basically, can be anything, as long as it's a const